runtime/vm/j9vm.tdf

```diff
@@ -494,18 +494,18 @@ TraceExit=Trc_VM_internalReleaseVMAccessNoMutex_Exit Overhead=1 Level=9 Template
 TraceEvent=Trc_VM_acquireExclusiveVMAccess_SettingResponsesExpected Group=exvmaccess Overhead=1 Level=1 Template="State is J9_XACCESS_HANDING_OFF, Setting repsonsesExpected to 1"
 
 TraceEntry=Trc_JNIinv_DestroyJavaVM_Entry NoEnv Overhead=1 Level=3 Template="JNIinv DestroyJavaVM javaVM=%p"
-TraceExit=Trc_JNIinv_DestroyJavaVM_alreadyShutdown_Exit NoEnv Overhead=1 Level=3 Template="JNIinv DestroyJavaVM shutdown already started" Exception
-TraceExit=Trc_JNIinv_DestroyJavaVM_failedAttach_Exit NoEnv Overhead=1 Level=3 Template="JNIinv DestroyJavaVM couldn't attach thread to shutdown" Exception
-TraceExit=Trc_JNIinv_DestroyJavaVM_verifyCurrentThreadAttached_Exit Obsolete NoEnv Overhead=1 Level=3 Template="JNIinv DestroyJavaVM current thread not attached. result=%d" Exception
+TraceExit-Exception=Trc_JNIinv_DestroyJavaVM_alreadyShutdown_Exit NoEnv Overhead=1 Level=3 Template="JNIinv DestroyJavaVM shutdown already started"
+TraceExit-Exception=Trc_JNIinv_DestroyJavaVM_failedAttach_Exit NoEnv Overhead=1 Level=3 Template="JNIinv DestroyJavaVM couldn't attach thread to shutdown"
+TraceExit-Exception=Trc_JNIinv_DestroyJavaVM_verifyCurrentThreadAttached_Exit Obsolete NoEnv Overhead=1 Level=3 Template="JNIinv DestroyJavaVM current thread not attached. result=%d"
 TraceEntry=Trc_JNIinv_protectedDestroyJavaVM_Entry NoEnv Overhead=1 Level=3 Template="protectedDestroyJavaVM"
 TraceEvent=Trc_JNIinv_protectedDestroyJavaVM_StartingThreadWait NoEnv Overhead=1 Level=3 Template="protectedDestroyJavaVM waiting for Java threads to stop"
 TraceEvent=Trc_JNIinv_protectedDestroyJavaVM_FinishedThreadWait NoEnv Overhead=1 Level=3 Template="protectedDestoryJavaVM all Java threads have stopped"
 TraceEvent=Trc_JNIinv_protectedDestroyJavaVM_vmCleanupDone NoEnv Overhead=1 Level=3 Template="protectedDestroyJavaVM protectedDestroyJavaVM vmCleanup complete"
 TraceEvent=Trc_JNIinv_protectedDestroyJavaVM_vmShutdownHookFired NoEnv Overhead=1 Level=3 Template="protectedDestroyJavaVM VM Shutting Down Hook Fired"
 TraceEvent=Trc_JNIinv_protectedDestroyJavaVM_HeapManagementShutdown NoEnv Overhead=1 Level=3 Template="protectedDestroyJavaVM GC HeapManagement Shutdown"
 TraceEvent=Trc_JNIinv_protectedDestroyJavaVM_vmShutdownDone NoEnv Overhead=1 Level=3 Template="protectedDestroyJavaVM vmShutdown returned"
-TraceEvent=Trc_JNIinv_protectedDestroyJavaVM_CallingExitHookSecondary NoEnv Overhead=1 Level=3 Template="protectedDestroyJavaVM calling exitHook secondary path" 
-TraceEvent=Trc_JNIinv_protectedDestroyJavaVM_terminateRemainingThreadsFailed NoEnv Overhead=1 Level=3 Template="protectedDestroyJavaVM terminateRemainingThreads failed" Exception
+TraceEvent=Trc_JNIinv_protectedDestroyJavaVM_CallingExitHookSecondary NoEnv Overhead=1 Level=3 Template="protectedDestroyJavaVM calling exitHook secondary path"
+TraceEvent=Trc_JNIinv_protectedDestroyJavaVM_terminateRemainingThreadsFailed NoEnv Overhead=1 Level=3 Template="protectedDestroyJavaVM terminateRemainingThreads failed"
```

Comment:
Shouldn;t this be TraceException instead of TraceEvent?

Reply:
That sounds reasonable: I'll update the commit.